crates/trade/src/cfd.rs

```diff
@@ -54,3 +58,61 @@ pub fn calculate_short_liquidation_price(leverage: Decimal, price: Decimal) -> D
 
     price * leverage / (leverage - Decimal::ONE)
 }
+
+// TODO: This was copied from ItchySats and adapted; we need tests for this!
```

Comment:
Yes please, I did not check the functions and trust you to verify this later on.